src/EcommerceStatsBundle/ChameleonSystemEcommerceStatsBundle.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
```

Comment:
All (new) core files need the license header.

Reply:
:heavy_check_mark: Added license header to classes